.orchestra/ci/parts/pipeline-create.sh

```diff
@@ -0,0 +1,77 @@
+#!/usr/bin/env bash
+set -euo pipefail
+
+# Pipeline creation
+# This script's responsibility is to trigger the execution of a pipeline of a
+# generic downstream project. The resulting ID is printed to stdout or a
+# non-zero exit code is returned
+#
+# Mandatory parameters:
+#
+# 1:
+#   File containing the list of changed files in the binary archives, each line
+#   is a different file path
+# COMPONENT_TARGET_BRANCH: target branch the CI has been triggered against
+#
+# DOWNSTREAM_PROJECT_URL:
+#   The base URL that will be used to create a pipeline for the project
+# DOWNSTREAM_PROJECT_TOKEN:
+#   A private token which will be used for authentication when making the
+#   request
+#
+# Optional parameters:
+#
+# DOWNSTREAM_PROJECT_TARGET_BRANCH:
+#   The target branch to use for the pipeline, defaults to 'master'
+# BINARY_ARCHIVES_HOOKS_FORCE:
+#   If set to '1' the hooks are going to be run regardless if there have been
+#   changes to the binary archives
```

Comment:
* `DOWNSTREAM_PROJECT_TARGET_BRANCH`: I'm ok dropping this
* `BINARY_ARCHIVES_HOOKS_FORCE`: I've added this so that we can trigger hooks without having to do changes like [this](https://github.com/revng/orchestra/commit/51b301d2efcbb508a20e6032f9f01c139289eb2b) every time we want to check if the hooks are working. While "unused" it has a purpose (forcing execution of binary archives hooks) and it's useful (similarly to `llvm_enable_pgo_lto`)

Reply:
OK.